conversion_utils/__main__.py

```diff
@@ -0,0 +1,77 @@
+import argparse
+from enum import StrEnum
+from pathlib import Path
+
+import pandas as pd
+
+
+def convert_toetsingskader_to_csv(path: Path) -> None:
+    """
+    Converts the second tab in the toetsingskader algoritmes excel to
+    csv.
+    """
+    df: pd.DataFrame = pd.read_excel(path, sheet_name=1)
+    df.rename(columns=df.iloc[0], inplace=True)
+    df.drop(df.index[0], inplace=True)
+
+    out_dir = Path("input")
+    out_dir.mkdir(parents=True, exist_ok=True)
+    df.to_csv(out_dir / "toetsingskader_algoritmes_v1.csv", index=False)
+
+
+def messy_date_to_iso8601(input: str) -> str:
+    try:
+        return pd.to_datetime(input).date().isoformat()
+    except (pd.errors.ParserError, ValueError):
+        return ""
+
+
+def split_algoritmeregister(path: Path) -> None:
+    """
+    Split algoritmeregister csv file into multiple csv files, where
+    each file contains entries with the same schema version.
+    """
+
+    # ------------
+    # Arguably this section should be removed and done in the yq transformations.
+
+    # Skip initial spaces because algoritmeregister contains random whitespaces/newline/tabs
+    # in fields.
+    df: pd.DataFrame = pd.read_csv(path, skipinitialspace=True)
+
+    # Cleanup messy dates to ISO 8601 date without timezone.
+    df["begin_date"] = df["begin_date"].apply(lambda x: messy_date_to_iso8601(x))
+    df["end_date"] = df["end_date"].apply(lambda x: messy_date_to_iso8601(x))
+
+    # ------------
+
+    out_dir = Path("input")
+    out_dir.mkdir(parents=True, exist_ok=True)
+
+    standard_versions = ("0.1.0", "0.4.0", "1.0.0")
+    for standard_version in standard_versions:
+        df_v = df[df["standard_version"] == standard_version]
+
+        # Save as for example: out_dir/algoritmeregister_v010.csv
+        save_path = out_dir / Path(f'algoritmeregister_v{standard_version.replace(".", "")}.csv')
+        df_v.to_csv(save_path, index=False)
+
+
+class Command(StrEnum):
```

Comment:
I would call it Action
```suggestion
class Action(StrEnum):
```